### ISSUE TITLE
New ideas

### ISSUE THREAD

**Issue by raulvillares** [raulvillares/3sacrowd#4]
Any idea to enhance the game? Let's talk about it!

**Comment by daffron** [comment]
Display level number?

**Comment by raulvillares** [comment]
Yeah, maybe in a "information area". For example:

```
Level 1/19
Movements: 22
Time: 1:54
```

**Comment by raulvillares** [comment]
It's a new feature now!

**Comment by raulvillares** [comment]
See #24

**Comment by flyer2403** [comment]
How about enlarging the squares apon hovering over them to improve the feel of the game?

**Comment by raulvillares** [comment]
Yes, I think it could look nicer. There is too much spacing between squares now. Do you feel like working on it? @flyer2403

**Comment by flyer2403** [comment]
Yes I'd like to work on this

**Comment by flyer2403** [comment]
I've created a pull request.
Please let me know if anything needs to be altered. I can also add similar functionality to the buttons.

**Comment by Apurva101** [comment]
Hi, we can add these features to enhance the game more:
1. A user can proceed to next level only when he has completed the current level with given constraints.
2.When a user tries to leave the site(game), we confirm him if he really wants to leave the site(added a confirmation dialog box)
Here's the link to my pull request in which I have added this feature.[https://github.com/raulvillares/3sacrowd/pull/52](url)

**Comment by UncleFifi** [comment]
how about adding an animation if the square cant be flipped. animation would shake the square. if admins approve this idea, I would like to work on this please.

**Comment by titivermeesch** [comment]
@UncleFifi That sounds good. I'll make a ticket for it

**Comment by OleDakotaJoe** [comment]
What about adding a help Dialog box, and button. So that when a new user tries to play, if they dont understand the basics fo gameplay automatically: they have a hint! 

I have created a PR with this feature (my first PR ever, so let me know if i did anything incorrectly)

https://github.com/raulvillares/3sacrowd/pull/65
